docs/content/release-notes/flink-1.13.md

```diff
@@ -0,0 +1,231 @@
+This behavior was inconsistent with all other deployment methods and ignored any limits on the CPU
+usage set via docker.
+
+#### Rework jemalloc switch to use an environment variable
+
+##### [FLINK-21034](https://issues.apache.org/jira/browse/FLINK-21034)
+
+The docker switch for disabling the jemalloc memory allocator has been reworked from a script
+argument to an environment variable called DISABLE_JEMALLOC. If set to "true" jemalloc will not be
+enabled.
+
+### Connectors
+
+#### Remove swift FS filesystem
+
+##### [FLINK-21819](https://issues.apache.org/jira/browse/FLINK-21819)
+
+The Swift FileSystem is no longer being actively developed and has been removed from the project and
+distribution.
+
+### Monitoring & debugging
+
+#### Introduce latency tracking state
+
+##### [FLINK-21736](https://issues.apache.org/jira/browse/FLINK-21736)
+
+State access latency metrics are introduced to track all kinds of keyed state access to help debug
+state performance. This feature is not enabled by default and could be turned on once
+setting `state.backend.latency-track.keyed-state-enabled` as true.
+
+#### Create backPressuredTimeMsPerSecond metric
+
+##### [FLINK-20717](https://issues.apache.org/jira/browse/FLINK-20717)
+
+Previously `idleTimeMsPerSecond` was defined as the time task spent waiting for either the input or
+the back pressure. Now `idleTimeMsPerSecond` excludes back pressured time, so if the task is back
+pressured it is not idle. The back pressured time is now measured separately as `backPressuredTimeMsPerSecond`.
+
+#### Enable log4j2 monitor interval by default
+
+##### [FLINK-20510](https://issues.apache.org/jira/browse/FLINK-20510)
+
+The Log4j support for updating the Log4j configuration at runtime has been enabled by default. The
+configuration files are checked for changes every 30 seconds.
+
+#### ZooKeeper quorum fails to start due to missing log4j library
+
+##### [FLINK-20404](https://issues.apache.org/jira/browse/FLINK-20404)
+
+The Zookeeper scripts in the Flink distribution have been modified to disable the Log4j JMX
+integration due to an incompatibility between Zookeeper 3.4 and Log4j 2. To re-enable this feature,
+remove the line in the `zookeeper.sh` file that sets `zookeeper.jmx.log4j.disable`.
+
+#### Expose stage of task initialization
+
+##### [FLINK-17012](https://issues.apache.org/jira/browse/FLINK-17012)
+
+Task's RUNNING state was split into two states: INITIALIZING and RUNNING. Task is INITIALIZING while
+state is initialising and in case of unaligned checkpoints, until all the in-flight data has been
+recovered.
+
+#### Support for CPU FlameGraphs in new web UI
+
+##### [FLINK-13550](https://issues.apache.org/jira/browse/FLINK-13550)
+
+Flink now offers Flamegraphs for each node in the job graph. Please enable this experimental feature
+by setting the respective configuration flag `rest.flamegraph.enabled`.
+
```

Comment:
```suggestion
#### Exception history being properly exposed through the REST API and UI

##### [FLINK-6042](https://issues.apache.org/jira/browse/FLINK-6042)

Flink exposes the exception history now through the REST API and the UI. The amount of most-recently handled exceptions that shall be tracked can be defined through `web.exception-history-size`.
```
I realized that [FLINK-6042](https://issues.apache.org/jira/browse/FLINK-6042) wasn't closed, yet, due to some follow-up work ([FLINK-21867](https://issues.apache.org/jira/browse/FLINK-21867)). I extracted `FLINK-21867` to be an independent task to be able to close `FLINK-6042` properly.
I propose to add it to the release notes as well.